samples/language/Qubit.qs

```diff
@@ -0,0 +1,40 @@
+/// # Sample
+/// Qubit
+///
+/// # Description
+/// Q# uses the `Qubit` primitive type to represent quantum state. A qubit represents 
+/// the smallest addressable physical unit in a quantum computer. Qubits are long-lived,
+/// and accumulates transformations to quantum states. A Q# program has no ability to
+/// introspect into the state of a qubit, and thus is entirely agnostic about what a
+/// quantum state is or on how it is realized. Rather, a program can call operations
+/// such as Measure to learn information about the quantum state of the computation.
+namespace MyQuantumApp {
+    
+    open Microsoft.Quantum.Intrinsic;
```

Comment:
```suggestion
```

Reply:
We can safely avoid opening the intrinsic namespace since it is automatically opened by the compiler.